test/runtime/write.c

```diff
@@ -12,6 +12,7 @@
 #include <sys/time.h>
 #include <sys/eventfd.h>
 #include <sys/stat.h>
+#include <unix/syscalls.h>
```

Comment:
This include shouldn't be needed anymore once `syscall(SYS_sync)` is replaced by `sync()`